ceremonies/src/tests.rs

```diff
@@ -810,8 +562,8 @@ fn claim_rewards_works() {
 				cid,
 				cindex,
 				meetup_index: 1,
-				account: account_id(&eve),
-				reason: ExclusionReason::TooFewOutgoingAttestations,
+				account: ferdie.clone(),
+				reason: ExclusionReason::NoVote,
```

Comment:
Before they had votes, as other people supplied claims for them, even though ferdie and eve did not send any attestations. Now the other attendees don't hand in votes for them anymore, which is why the error changed.